azkaban-execserver/src/package/conf/azkaban.properties

```diff
@@ -25,6 +25,12 @@ executor.flow.threads=30
 jetty.connector.stats=true
 executor.connector.stats=true
 
+
 # uncomment to enable inmemory stats for azkaban
 #executor.metric.reports=true
-#executor.metric.milisecinterval.default=60000
\ No newline at end of file
+#executor.metric.milisecinterval.default=60000
+
+#ConnectorBufferSizes
+headerBufferSize=4096
```

Comment:
Could we make this name to be 
"jetty.connector.*"

This way it is more clear the setting is regarding jetty.connectors,
and also fits with the "jetty.connector.stats" specification above